src/viam/sdk/services/mlmodel/private/proto.cpp

```diff
@@ -24,9 +26,11 @@ namespace mlmodel_details {
 
 namespace {
 
+namespace gp = ::google::protobuf;
```

Comment:
Just for shorter lines, you can ignore all diffs of `google::protobuf::` to `gp::`.